defs.h

```diff
@@ -246,6 +255,35 @@ struct bounds {
   double min_alt;	/* -unknown_alt => invalid */
 };
 
+struct PositionRad; // forward declare
+struct PositionDeg
+{
+  PositionDeg() = default;
+  explicit(false) inline PositionDeg(const PositionRad& posr); /* converting ctor */
```

Comment:
No, they must not be explicit.  The "explicit(false)" is trying to tell clang-tidy to dummy up about
 >single-argument constructors must be marked explicit to avoid unintentional implicit conversions [google-explicit-constructor]

If they are explicit then the implicit conversions fail, e.g.:
```
arcdist.cc:64:41: error: could not convert ‘arcpt2->Waypoint::position()’ from ‘PositionDeg’ to ‘PositionRad’
   64 |           dist = gcdist(arcpt2->position(), waypointp->position());
      |                         ~~~~~~~~~~~~~~~~^~
      |                                         |
      |                                         PositionDeg
```